.tool-versions

```diff
@@ -1,8 +1,8 @@
 golang 1.21.7
 mockery 2.42.2
-nodejs 16.16.0
-pnpm 8.11.0
-postgres 13.3
+nodejs 20.13.1
+pnpm 8.15.8
+postgres 15.1
```

Comment:
when did we move to postgres 15.1  o_O... I guess it looks like we're on 15.x in most of CI but what a big jump

Reply:
nops are going to have so much fun when someone starts writing some new migration that won't run on older pg O_o... still kind of shocked and wondering what the process is even for db upgrades as it's not a trivial change depending on how much existing data is involved

Reply:
.tool-versions is just used for development

Reply:
but yeah, at some point if we ever use a feature that isn't on all versions we'll need our NOPs to do a database upgrade.